openfga/v1/authzmodel.proto

```diff
@@ -80,10 +82,27 @@ message RelationTypeInfo {
 
 message Metadata {
   map<string, RelationMetadata> relations = 1;
+  string file = 2 [(validate.rules).string = {
```

Comment:
I didn't think we're going to be allowing relations of a type to be split across files and modules, are we? If not, these need to be moved up to into the `TypeDefinition` .

Reply:
We are, types can be extended in other files

Reply:
Yeah it wasn't initially planned but we decided to include it after feedback that it would be a desired feature.

We split across `TypeDefinition` and `Relation` so that relations that are from an extended type can set their module/file data. Relations that are in the same source file as the type will not include this metadata